Gateway/Request/OneclickAuthorizationDataBuilder.php

```diff
@@ -79,6 +91,13 @@ public function build(array $buildSubject)
             }
         }
 
+        // if installments is set add it into the request
+        if ($payment->getAdditionalInformation(AdyenOneclickDataAssignObserver::NUMBER_OF_INSTALLMENTS) &&
+            $payment->getAdditionalInformation(AdyenOneclickDataAssignObserver::NUMBER_OF_INSTALLMENTS) > 0
```

Comment:
duplicated lines?

Reply:
copied it over from ccAuthorisationBuilder. We could probably simplify it although it could be that > 0 could lead to an notice not sure.

Reply:
okay keep it as it is for now